client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -221,7 +221,7 @@ export default React.createClass( {
 				domain.googleAppsSubscription &&
 				domain.googleAppsSubscription.pendingUsers &&
 				domain.googleAppsSubscription.pendingUsers.length !== 0 );
-		return pendingDomains.length !== 0 && <PendingGappsTosNotice key="pending-gapps-tos-notice" site={ this.props.selectedSite } domains={ pendingDomains } section="domain-management" />;
+		return pendingDomains.length !== 0 && <PendingGappsTosNotice key="pending-gapps-tos-notice" siteSlug={ this.props.selectedSite.slug } domains={ pendingDomains } section="domain-management" />;
```

Comment:
Yes and no - it can be passed `false` from `CurrentSite` (which, despite its name, is also shown when browsing all your sites), but the rule for showing this notice is not whitelisted in that context. Nevertheless, to be on the safe side, I've added the check - good catch!